modules/ROOT/pages/oracle/oracle-ebs-connector.adoc

```diff
@@ -159,11 +155,17 @@ For PL/SQL functions, the connector supports the following out of the box, along
 *** Update Org Contact +
 *** Update Org Contact Role +
 
-== To Connect in Design Center
+== Connect in Design Center
 
-. Click a trigger. You can use this connector as the trigger, or an HTTP Listener or Scheduler trigger. For
+. Select a trigger. You can use this connector as the trigger, or an HTTP Listener or Scheduler trigger. For
```

Comment:
GLOBAL and in template, if any list items are still longer than one sentence or fragment:
```suggestion
. Select a trigger. 
You can use this connector as the trigger, or you can use an HTTP Listener or Scheduler trigger. For
```